src/interfaces/IREP15.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
+
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+/// Note: the ERC-165 identifier for this interface is 0xba63ebbb.
+interface IREP15 is IERC165 {
+  /// @dev This emits when a context is updated by any mechanism.
+  event ContextUpdated(bytes32 indexed ctxHash, address indexed controller, uint64 detachingDuration);
+  /// @dev This emits when a token is attached to a certain context by any mechanism.
+  event ContextAttached(bytes32 indexed ctxHash, uint256 indexed tokenId);
+  /// @dev This emits when a token is requested to detach from a certain context by any mechanism.
+  event ContextDetachmentRequested(bytes32 indexed ctxHash, uint256 indexed tokenId);
+  /// @dev This emits when a token is detached from a certain context by any mechanism.
+  event ContextDetached(bytes32 indexed ctxHash, uint256 indexed tokenId);
+  /// @dev This emits when a user is assigned to a certain context by any mechanism.
+  event ContextUserAssigned(bytes32 indexed ctxHash, uint256 indexed tokenId, address indexed user);
+  /// @dev This emits when a token is (un)locked in a certain context by any mechanism.
+  event ContextLockUpdated(bytes32 indexed ctxHash, uint256 indexed tokenId, bool locked);
+  /// @dev This emits when the ownership delegation is started by any mechanism.
+  event OwnershipDelegationStarted(uint256 indexed tokenId, address indexed delegatee, uint64 until);
+  /// @dev This emits when the ownership delegation is accepted by any mechanism.
+  event OwnershipDelegationAccepted(uint256 indexed tokenId, address indexed delegatee, uint64 until);
+  /// @dev This emits when the ownership delegation is stopped by any mechanism.
+  event OwnershipDelegationStopped(uint256 indexed tokenId, address indexed delegatee);
+
+  /// @notice Gets the longest duration the detaching can happen.
+  function maxDetachingDuration() external view returns (uint64);
```

Comment:
```suggestion
  function MAX_DETACHING_DURATION() external view returns (uint64);
```

Reply:
do not modify the interface, cc @ducthotran2010